test/Feature/BFSSearcherAndDFSSearcherInterleaved.c

```diff
@@ -0,0 +1,42 @@
+// RUN: %llvmgcc %s -emit-llvm -g -c -o %t.bc
+// RUN: rm -rf %t-bfs.klee-out
+// RUN: rm -rf %t-dfs.klee-out
+// RUN: rm -rf %t-bfs-dfs.klee-out
+// RUN: rm -rf %t-dfs-bfs.klee-out
+// RUN: %klee -libc=uclibc -output-dir=%t-bfs.klee-out -search=bfs %t.bc >%t-bfs.out
```

Comment:
You should remove `-libc-uclibc`. This test doesn't need it (`printf()` can be called as an external function because it has concrete arguments). This is the cause of the TravisCI failure for macOS.